src/streams/CreateStreamScreen.js

```diff
@@ -20,13 +22,21 @@ type Props = $ReadOnly<{|
 export default function CreateStreamScreen(props: Props): Node {
   const auth = useSelector(getAuth);
   const ownEmail = useSelector(getOwnEmail);
+  const getStreams = useSelector(getStreamsByName);
```

Comment:
`getStreams` isn't the right name for this value, which is a `Map<string, Stream>`. It makes it sound like the value is a function that, when called, will "get streams".

Let's use `streamsByName`. That's a more natural name for the result of using the `getStreamsByName` selector.

Reply:
Sure @chrisbobbe!, I'll change the name.